### DISCUSSION TITLE
v1.12.16

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2180] [分享 / Share]
📌**公告**

长期合作开发者 @Coulomb-G 因个人发展原因, 即日起暂停对脚本相关功能的开发, 感谢他这三年来的付出~ (2018-09-05 ~ 2021-08-21)
- `界面翻译`将暂停更新.
- `夜间模式`将降低更新速度.

✨**新增**

- `简化直播间`适配新版直播播放器, 支持屏蔽表情特效. (#2116)
- `动态过滤器`设置的过滤关键词也将作用于`简化首页`中显示的推荐视频里. (#2087)
- SpinQuery API 支持在页面未聚焦时暂停运行, 这有助于改善一些组件的性能, 虽然是根据 #2078 的讨论做的, 不过并没能解决 #2078 的问题. 即便是页面在前台了, 非用户鼠标点击触发的全屏请求依然有几率被浏览器拒绝.

🐛**改进/修复**

- 夜间模式更新: (#2136)
  - 粉丝勋章墙 (#2025)
  - 专栏勋章 (#2083)
  - "危险行为"标识 (#2094)
  - 番剧点评 (#1971, #2072, #1143)
  - 个人资料 (#2105)
  - 视频投票弹窗 (#2115)
  - 观看人数提示 (#2162)
- `简化评论区`适配评论区样式变动.
- 个人空间的动态搜索适配了动态相关功能. (#2137)
- 降低了侧栏图标的 `z-index`. (#2095)
- 修复直播弹幕设置被排行榜遮挡. (#2097)
- 修复`评论翻译`导致评论回复间距异常. (#2159)
- 修复`高分辨率图片`导致订阅合集封面尺寸错误. (#2158)
- 修复`自定义顶栏`无法显示未读消息数目. (#2171)
- 排除轻视频链接. (#2156)
- 设置面板停靠在右侧时不隐藏首页的联系客服. (#1929)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.16'>v1.12.16</a>.</em>

**Comment by MasterOfStar** [comment]
Thanks  @Coulomb-G

**Comment by ilharp** [comment]
感谢 Coulomb 佬一直以来的贡献啊，大佬辛苦了
